### ISSUE TITLE
Can't changue valet domain 

### ISSUE THREAD

**Issue by flick36** [cpriego/valet-linux#42]
Just reporting a bug when i try `valet domain app` i get:

```
PHP Warning:  file_put_contents(/etc/NetworkManager/dnsmasq.d/valet): failed to open stream: Permission denied in /home/flick/.config/composer/vendor/cpriego/valet-linux/cli/Valet/Filesystem.php on line 176
```

I'm on Arch(Manjaro)

**Comment by cpriego** [comment]
Hi @flick36, you are right. I'm fixing this right now.

**Comment by cpriego** [comment]
Fixed in version [2.0.10](https://github.com/cpriego/valet-linux/releases/tag/v2.0.10)
